ui/apps/platform/src/services/ComplianceIntegrationService.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'services/instance';
+
+import { complianceV2Url } from './ComplianceCommon';
+
+const complianceIntegrationsBaseUrl = `${complianceV2Url}/integrations`;
+
+export type ComplianceIntegration = {
+    id: string;
+    version: string;
+    clusterId: string;
+    clusterName: string;
+    namespace: string;
+    statusErrors: string[];
```

Comment:
What is pro and con to include `operatorInstalled: boolean` property?
https://github.com/stackrox/stackrox/blob/master/proto/api/v2/compliance_integration_service.proto#L20

Is it redundant because response is filtered to include only clusters which have operator installed?

Reply:
Actually the response will include clusters that don't have operator installed but those clusters include a `statusError` making the cluster appear unhealthy. Maybe `operatorInstalled` was added to differentiate the two? Seems like a good topic of discussion considering I don't recall hearing anything about that field.